CHANGELOG.md

```diff
@@ -40,7 +40,7 @@ This section is for maintaining a changelog for all breaking changes for the cli
 ### Removed
 
 ### Fixed
-
+- Fixed error when deserializing a normalizer without 'type' [#1111](https://github.com/opensearch-project/opensearch-java/pull/1111)
```

Comment:
The PR number should be surrounded by parenthesis and a blank line before the next heading please
```suggestion
- Fixed error when deserializing a normalizer without 'type' ([#1111](https://github.com/opensearch-project/opensearch-java/pull/1111))

```